Runtime/Native/Windows/NativeClient.cs

```diff
@@ -323,13 +323,20 @@ private string GetDefaultPathToCrashpadHandler()
         {
             const string crashpadHandlerName = "crashpad_handler.dll";
             const string pluginDir = "Plugins";
-            string architecture = IntPtr.Size == 8 ? "x86_64" : "x86";
-
-            string pluginPath = Path.Combine(pluginDir, architecture);
-            string pluginHandlerPath = Path.Combine(pluginPath, crashpadHandlerName);
+            string[] availablePlugins = new string[2] { "x86_64", "x86" };
+            var pluginDirectoryPath = Path.Combine(Application.dataPath, pluginDir);
+            var pluginDirectory = new DirectoryInfo(pluginDirectoryPath);
+            if (!pluginDirectory.Exists)
+            {
+                return string.Empty;
```

Comment:
small nit: wouldn't `null` be more appropriate here?

Reply:
it's a better idea. thx